process/conf.yaml.example

```diff
@@ -29,6 +29,7 @@ instances:
 #                   above the second one, the process check will return CRITICAL.
 #     In this example, process check will return OK for 3 to 5 process. WARNING for 1, 2, 6, 7 processes and Critical below 1 or above 7.
 #     CRITICAL is always dominant in case of overlapping.
+#    collect_children: BOOLEAN. If true, the check will also collect metrics from all child processes of a matched process. Default to false.
```

Comment:
We could add a warning stating that the collection is recursive and depending on the use case might take some time.